pages/home/ThankYouModal.jsx

```diff
@@ -1,18 +1,23 @@
 var React = require('react');
 var Modal = require('../../components/modal.jsx');
+var FormattedMessage = require('react-intl').FormattedMessage;
 
 var ThankYouModal = React.createClass({
+  contextTypes: {
+    intl: React.PropTypes.object
+  },
   render: function() {
   // we can't preset the message sharing on Facebook
   var facebookShare = "https://www.facebook.com/sharer/sharer.php?u=" + encodeURIComponent("https://learning.mozilla.org");
-  var twitterShare = "https://twitter.com/home?status=" + encodeURIComponent("I love to #TeachTheWeb! https://learning.mozilla.org");
+  var twitterShareMessage = this.context.intl.formatMessage({id: 'twitter_share_message'}, {learningURL: "https://learning.mozilla.org" });
+  var twitterShare = "https://twitter.com/home?status=" + encodeURIComponent(twitterShareMessage);
     return (
-      <Modal modalTitle="Thanks for your interest!" className="modal-signup" hideModal={this.props.hideModal}>
-        <p>We appreciate your commitment to keeping the web open, accessible and ours.</p>
-        <p><strong>Share and tell your friends</strong></p>
+      <Modal modalTitle={this.context.intl.formatMessage({id: 'thanks_for_your_interest'})} className="modal-signup" hideModal={this.props.hideModal}>
+        <p><FormattedMessage id='thank_you_modal_message' /></p>
+        <p><strong><FormattedMessage id='thank_you_modal_share_message' /></strong></p>
         <div className="social-share">
-          <a href={facebookShare} className="facebook"><i className="fa fa-facebook"></i>Facebook</a>
```

Comment:
There was a history of disagreement between localizers in different community about this matter, so I want to leave them to make the decision here...